### DISCUSSION TITLE
generate_answer 'NoneType' object has no attribute 'encode'

### DISCUSSION THREAD

**Discussion by ahoepf** [innightwolfsleep/llm_telegram_bot#121] [Q&A]
Hello everyone, I am encountering the following error "user_text Test
generate_answer 'NoneType' object has no attribute 'encode'
cannot unpack non-iterable NoneType object
Exception in thread Thread-6 (thread_get_message):
Traceback (most recent call last):
  File "/home/alberth/miniconda3/envs/textgen/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/alberth/miniconda3/envs/textgen/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/alberth/deeplearning/textgen/text-generation-webui/extensions/telegram_bot/TelegramBotWrapper.py", line 765, in thread_get_message
    raise e
  File "/home/alberth/deeplearning/textgen/text-generation-webui/extensions/telegram_bot/TelegramBotWrapper.py", line 746, in thread_get_message
    answer, system_message = self.generate_answer(
TypeError: cannot unpack non-iterable NoneType object
" and unfortunately, I am unable to progress. I have configured everything according to the instructions. I am using the latest checked out version of text-generation-webui.

**Comment by innightwolfsleep** [comment]
Hi! I just made clear install of oobaboba and telegram_bot but I can't repeat your bug. 

I am using llama7b4bit model and flags:
"--chat --gpu-memory 6 --cpu-memory 6 --model llama7b4bit --sdp-attention --api --extensions telegram_bot"

Can you share details about you command flags, prompt and config? I see you are not alone with this bug https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/issues/119

**Reply by ahoepf** [reply]
Hi, thank you for response,  i'm using this flags:
"--api --extensions superbooga silero_tts whisper_stt telegram_bot"
and when i tried your flags, no errors happened X-)

**Comment by sakhavhyand** [comment]
I've got the same symptoms today when playing with a new model.
The model auto-load with AutoGPTQ : no problem, I load it with ExLlama_HF : problem appear, load it again with ExLlama : problem disappear.
During all of this testing, everything was working fine through the WebUI.

It's strange because I have other models loaded via ExLlama_HF and they work well in the telegram integration.
The model causing problem in my case is [TheBloke/MythoMax-L2-13B-GPTQ](https://huggingface.co/TheBloke/MythoMax-L2-13B-GPTQ)
